proxy/healthy_endpoints.go

```diff
@@ -21,7 +21,8 @@ func (h *healthyEndpoints) filterHealthyEndpoints(endpoints []routing.LBEndpoint
 	filtered := make([]routing.LBEndpoint, 0, len(endpoints))
 	for _, e := range endpoints {
 		if p < e.Metrics.HealthCheckDropProbability() {
-			/* drop */
+			ctx.Logger().Infof("dropping endpoint %q due to passive health check: p=%0.2f, dropProbability=%0.2f",
+				e.Host, p, e.Metrics.HealthCheckDropProbability())
```

Comment:
To print `e.Metrics.HealthCheckDropProbability()` you need to capture its value as it can change between check and log.